src/app/ApplicationTemplate.Presentation/ViewModels/MenuViewModel.cs

```diff
@@ -52,6 +59,7 @@ public enum Section
 			.Select(state =>
 			{
 				var vmType = state.GetViewModelType();
+				SelectedIndex = _viewModelsWithBottomMenu.IndexOf(vmType);
```

Comment:
I'm not sure to understand why it's needed?

Reply:
Like shouldn't the twoway binding you're doing in the xaml be enough?

Reply:
This line is to update the SelectedIndex when changing tab without clicking on the bottom menu buttons. 
For example when we logout we do it from the "Profile" tab, and when login again we land on the "Jokes" tab, but since we didn't click on the "Jokes" button in the menu, without this line the "Profile" button would appear as selected.